docs/verify_cri.md

```diff
@@ -0,0 +1,100 @@
+# Verify pouch CRI by the cri-tools
```

Comment:
Could you change the file name to be  `pouch_with_cri_verification.go` to keep consistent?
And also change the head title of this file.